test/testSessionState.js

```diff
@@ -6,7 +6,7 @@ const STATE = require('./../SessionStateEnum.js');
 const XETHRU_STATE = require('./../SessionStateXethruEnum.js');
 const OD_FLAG_STATE = require('./../SessionStateODFlagEnum.js');
 const DOOR_STATE = require('./../SessionStateDoorEnum.js');
-const SessionState = require('./../SessionState.js');
+const StateMachine = require('./../StateMachine.js');
```

Comment:
Could you please rename this file to `testStateMachine.js` to match the file that it's testing?

Reply:
Can do!

Reply:
yup!